pkg/sql/logictest/testdata/logic_test/pg_catalog

```diff
@@ -1055,6 +1056,7 @@ FROM pg_catalog.pg_roles
 ORDER BY rolname
 ----
 oid         rolname   rolpassword  rolvaliduntil  rolconfig
+823966177   admin     ·            NULL           {}
```

Comment:
This seems wrong - why is rolpassword = `.`? Shouldn't it also be ****..?

Reply:
I forgot to check postgres. It also populates the password field for roles even if they can't login. Fixed.